coordinator/src/main/scala/filodb/coordinator/RemoteActorPlanDispatcher.scala

```diff
@@ -0,0 +1,35 @@
+package filodb.coordinator
+
+import monix.eval.Task
+import monix.execution.Scheduler
+import monix.reactive.Observable
+
+import filodb.core.store.ChunkSource
+import filodb.query.{QueryResponse, StreamQueryResponse}
+import filodb.query.exec.{ExecPlanWithClientParams, PlanDispatcher}
+
+// RemoteActorPlanDispatcher is different from ActorPlanDispatcher because at the moment
+// of its instantiation it cannot have a valid ActorRef since it's referring to a machine
+// that is not part of the local FiloDB cluster.
+// However, when the dispatch call happens actor ref is created and normal ActorPlanDispatcher
+// is instantiated and utilized to fulfill dispatch.
+case class RemoteActorPlanDispatcher(path: String, clusterName: String) extends PlanDispatcher {
+
+  override def dispatch(
+     plan: ExecPlanWithClientParams, source: ChunkSource
+  )(implicit sched: Scheduler): Task[QueryResponse] = {
+    val serialization = akka.serialization.SerializationExtension(ActorSystemHolder.system)
+    val deserializedActorRef = serialization.system.provider.resolveActorRef(path)
```

Comment:
Maybe I am missing something. The path is of the remote filodb cluster. Wondering if the actor ref resolve will fail since remote machine is not part of filodb cluster ?

Reply:
Sycn'd offline on this.